python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -193,9 +195,9 @@ def get_dbt_multi_asset_args(
             key=asset_key,
             dagster_type=Nothing,
             io_manager_key=io_manager_key,
+            metadata=metadata,
             description=default_description_fn(node_info, display_raw_sql=False),
             is_required=False,
-            metadata=default_metadata_fn(node_info),
```

Comment:
Originally metadata was sourced from dbt manifest, but I think this metadata was actually dropped from the final asset and not passed through.

We could change the approach in this PR and instead of using metadata from the decorator source it from the manifest. The original dbt decorator included both options, essentially, by providing a function `node_info_to_definition_metadata_fn`. I'd prefer not to add that here, but we could I guess combine metadata from the decorator with metadata from the manifest automatically.